packages/wonder-blocks-labeled-field/src/index.ts

```diff
@@ -1,3 +1,3 @@
 import LabeledField from "./components/labeled-field";
 
-export default LabeledField;
+export {LabeledField};
```

Comment:
I was thinking if we ever add other components to this package, we'd have to change it to a named export so we could export multiple components, so I am changing it now! Any concerns with this?

Reply:
Sounds great to me! also like named exports so people have to use the right name (vs a default export where they can name it in any way).